src/paperless_mail/admin.py

```diff
@@ -105,5 +106,31 @@ class MailRuleAdmin(admin.ModelAdmin):
     ordering = ["order"]
 
 
+class ProcessedMailAdmin(admin.ModelAdmin):
+    class Meta:
+
+        model = ProcessedMail
+        fields = "__all__"
+
+    list_display = ("rule", "processed", "status", "subject", "received")
```

Comment:
```suggestion
    list_display = ("uid", "rule", "processed", "status", "subject", "received")
```

I think this would make more sense in admin so the uid is the unique thing that's clickable
e.g. 
<img width="549" alt="Screenshot 2023-03-01 at 1 44 18 AM" src="https://user-images.githubusercontent.com/4887959/222102453-ebc06283-0520-4378-9850-d9258fb0cd3b.png">
instead of
<img width="1013" alt="Screenshot 2023-03-01 at 1 28 04 AM" src="https://user-images.githubusercontent.com/4887959/222098485-1322a0a2-fd97-4298-a777-63881c193de2.png">